src/common/kernel-consumer/kernel-consumer.h

```diff
@@ -32,7 +32,7 @@ int lttng_kconsumer_get_consumed_snapshot(struct lttng_consumer_stream *stream,
 int lttng_kconsumer_recv_cmd(struct lttng_consumer_local_data *ctx,
 		int sock, struct pollfd *consumer_sockpoll);
 ssize_t lttng_kconsumer_read_subbuffer(struct lttng_consumer_stream *stream,
-		struct lttng_consumer_local_data *ctx);
+		struct lttng_consumer_local_data *ctx, bool *rotated);
```

Comment:
Since we are using the bool type, I think we should `#include <stdbool.h>`. I know it's being included by other headers. I prefer when the headers for the types used being explicitly included.